commands/link.js

```diff
@@ -0,0 +1,158 @@
+var Command = require('ronin').Command;
+var Q = require('q');
+var fs = require('fs-extra');
+var path = require('path');
+var readFile = Q.denodeify(fs.readFile);
+var readDir = Q.denodeify(fs.readdir);
+var remove = Q.denodeify(fs.remove);
+var inquirer = require('inquirer');
+var utils = require('../lib/util');
+var createLink = require('../lib/createLink');
+var chalk = require('chalk');
+
+module.exports = Command.extend({
+    help: function () {
+        var title = chalk.bold;
+        var d = chalk.gray;
+        var r = '\n  ' + title('Usage') + ': bb ' + this.name + ' [OPTIONS]';
+        r += '\n\t Symlinks source directory to target.\n\t  If --lp path is set, target will be: ';
+        r += '\n\t    ' +  d('{lp path}/launchpad-bundles/static/launchpad/{bundle}/widgets/{package name}');
```

Comment:
That is exactly the point. It is hardcoded because launchpad is using different paths for their widgets. It also depends on bundle(universal, banking etc)
Otherwise, I wouldn't have another flag just for launchpad